doc/references.bib

```diff
@@ -896,3 +896,12 @@ @article{kelleher2009generating
   journal={arXiv preprint arXiv:0909.2331},
   year={2009}
 }
+
```

Comment:
I'd strongly encourage to also cite this paper:

@article{rahimi2015can,
  title={What can quantum optics say about computational complexity theory?},
  author={Rahimi-Keshari, Saleh and Lund, Austin P and Ralph, Timothy C},
  journal={Physical review letters},
  volume={114},
  number={6},
  pages={060501},
  year={2015},
  publisher={APS}
}

Reply:
We can cite this in thewalrus. We have not really used it in SF.